shaman/controllers/repos/projects.py

```diff
@@ -55,12 +55,17 @@ def index_post(self):
             url=request.json.get("url", ""),
         )
         repo.update_from_json(update_data)
+        archs = request.json.get("archs", [])
```

Comment:
I just figured it was safer this way and not hurting anything. Also, I didn't have to update as many tests this way.